src/sage/databases/oeis.py

```diff
@@ -2019,7 +2019,7 @@ def test_compile_sage_code(self):
             RuntimeWarning: This sequence is dead: ...
             True
         """
-        if self.is_dead():
+        if self.is_dead(warn_only=True):
```

Comment:
This does not look right: `self.is_dead(warn_only=True)` will always return `None`.  The bug does not surface, because A000017 does not have sage code.